services/QuillLMS/client/app/bundles/Tools/components/evidenceHomeSection.tsx

```diff
@@ -0,0 +1,74 @@
+          <div className="divider-tab" />
+          <p className="description">Students read a nonfiction text and build their comprehension through writing prompts, supporting a series of claims with evidence sourced from the text. Quill challenges students to write responses that are precise, logical, and based on textual evidence, with Quill coaching the student through custom, targeted feedback on each revision so that students strengthen their reading comprehension and hone their writing skills.</p>
+          <a className="q-button bg-white text-quillteal" href={`${process.env.DEFAULT_URL}/tools/evidence`} rel="noopener noreferrer" target="_blank">Learn more about Quill Reading for Evidence</a>
+        </div>
+        <div className="image-container">
+          <img alt="A screenshot of an example Reading for Evidence prompt" className={`evidence-image ${evidenceImageUrl === IMG_URLS[11] ? '' : INACTIVE}`} src={IMG_URLS[11]} />
+          <img alt="A screenshot of an example Reading for Evidence prompt" className={`evidence-image ${evidenceImageUrl === IMG_URLS[10] ? '' : INACTIVE}`} src={IMG_URLS[10]} />
+          <img alt="A screenshot of an example Reading for Evidence prompt" className={`evidence-image ${evidenceImageUrl === IMG_URLS[9] ? '' : INACTIVE}`} src={IMG_URLS[9]} />
+          <img alt="A screenshot of an example Reading for Evidence prompt" className={`evidence-image ${evidenceImageUrl === IMG_URLS[8] ? '' : INACTIVE}`} src={IMG_URLS[8]} />
+          <img alt="A screenshot of an example Reading for Evidence prompt" className={`evidence-image ${evidenceImageUrl === IMG_URLS[7] ? '' : INACTIVE}`} src={IMG_URLS[7]} />
+          <img alt="A screenshot of an example Reading for Evidence prompt" className={`evidence-image ${evidenceImageUrl === IMG_URLS[6] ? '' : INACTIVE}`} src={IMG_URLS[6]} />
+          <img alt="A screenshot of an example Reading for Evidence prompt" className={`evidence-image ${evidenceImageUrl === IMG_URLS[5] ? '' : INACTIVE}`} src={IMG_URLS[5]} />
+          <img alt="A screenshot of an example Reading for Evidence prompt" className={`evidence-image ${evidenceImageUrl === IMG_URLS[4] ? '' : INACTIVE}`} src={IMG_URLS[4]} />
+          <img alt="A screenshot of an example Reading for Evidence prompt" className={`evidence-image ${evidenceImageUrl === IMG_URLS[3] ? '' : INACTIVE}`} src={IMG_URLS[3]} />
+          <img alt="A screenshot of an example Reading for Evidence prompt" className={`evidence-image ${evidenceImageUrl === IMG_URLS[2] ? '' : INACTIVE}`} src={IMG_URLS[2]} />
+          <img alt="A screenshot of an example Reading for Evidence prompt" className={`evidence-image ${evidenceImageUrl === IMG_URLS[1] ? '' : INACTIVE}`} src={IMG_URLS[1]} />
+          <img alt="A screenshot of an example Reading for Evidence prompt" className={`evidence-image ${evidenceImageUrl === IMG_URLS[0] ? '' : INACTIVE}`} src={IMG_URLS[0]} />
```

Comment:
I think since these are all the same except their index, it would probably be best just to iterate through an array of numbers and have it map to a shared component - something like:

```
IMG_URLS.map(url => (
  <img alt="A screenshot of an example Reading for Evidence prompt" className={`evidence-image ${evidenceImageUrl === url ? '' : INACTIVE}`} src={url} />
)
```

Also, as far as the img alt text goes - I think we should probably either describe these in more detail (with the text of the student's submission and feedback) or just leave it as an empty string.

Reply:
good catch, I'll make that update